packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.css

```diff
@@ -91,6 +91,12 @@
     grid-area: options;
 }
 
+.formErrors {
+    composes: section;
+    grid-area: errors;
+    color: rgb(var(--venia-global-color-error));
```

Comment:
Hm, I had hoped `FormError` would be one of those things that we basically never needed to restyle. I get that you'd want to give it a `grid-area` here, so that makes sense, but changing its text color seems like something that would happen globally or not at all.

Reply:
TBH I added color before, when it was not using form error and just forgot to remove this. If I change it'll require a reapproval :D What fun.